Gopkg.toml

```diff
@@ -0,0 +1,192 @@
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
```

Comment:
Hmm, shouldn't we add go-bindata to the required list? AFAIU the Gopkg.toml documentation, generators are what the directive is supposed to be used for.

Reply:
`required` add `github.com/jteeuwen/go-bindata/` to vendor folder, seems not wanted.

Reply:
Hmm okay. I guess we could expect people to call the binary from the vendor folder, but that seems  awkward to use.